lib/messages.c

```diff
@@ -288,6 +289,7 @@ msg_deinit(void)
 
 static GOptionEntry msg_option_entries[] =
 {
+  { "sdebug",            'r',         0, G_OPTION_ARG_NONE, &startup_debug_flag, "Enable startup debug message", NULL},
   { "verbose",           'v',         0, G_OPTION_ARG_NONE, &verbose_flag, "Be a bit more verbose", NULL },
```

Comment:
please use a longer option name, like --startup-debug

Also the help message would need some more clarification, "Enable debug logging during startup" or something.